Automation/create_ip_address.ps1

```diff
@@ -0,0 +1,13 @@
+## Command to create a public ip ##
+
+# Take a user input 
+param(
+        [string] [Parameter(Mandatory=$true)] $rgname, # Name of the resource group
+        [string] [Parameter(Mandatory=$true)] $location, # Location of Azure resources
+        [string] [Parameter(Mandatory=$true)] $publicIp, # Name of the azure public address
+        [string] [Parameter(Mandatory=$true)] $sku, # Type of SKU, Standard or Basic
+        [string] [Parameter(Mandatory=$true)] $allocation # AllocationMethod, Static or Dynamic
+      )
+
+# Creates a new Public IP Address based on the params.
+New-AzPublicIpAddress -ResourceGroupName $rgname -Name $publicIp -Location $location -AllocationMethod $allocation -SKU $sku
```

Comment:
We don't want there to be any user interaction/prompting, so the directions should say to add all of it on the same line, like 
`./Automation/create_ip_address.ps1 -rgname test-natanaels-pr-ly -location "uksouth" -publicIp "ip-address-test-ly" -sku Basic -allocation Dynamic`

Also consider doing some checks on the resource group, have it create a new rg if that one doesn't already exist
You will get errors if you try to create an IP address with a location that doesn't match the RG, so you can use the same location for both.